ckanext/canada/view.py

```diff
@@ -579,6 +596,79 @@ def ckanadmin_publish_datasets():
     return h.redirect_to('canada.ckanadmin_publish')
 
 
+@canada_views.route('/ckan-admin/job-queue', methods=['GET'])
+def ckanadmin_job_queue():
+    """
+    Lists all of the queued and running jobs.
+    """
+    try:
+        jobs = get_action('job_list')({}, {})
+    except NotAuthorized:
+        abort(403, _('Not authorized to see this page'))
+
+    warning = False
+    if jobs and datetime.strptime(jobs[0]['created'], '%Y-%m-%dT%H:%M:%S') < (datetime.now() - timedelta(minutes=18)):
+        warning = True
+
+    job_list = []
+    mystery_job_count = 0
+    for job in jobs[:25]:
```

Comment:
give this constant al ALLCAPS_NAME and put it at the module level, or better yet make it a config option

Reply:
which const? `mystery_job_count`? I decided not to use that in the output, so we can probably just remove it from the code here

Reply:
I mean the `25` cut-off

Reply:
In a constant now, will make an option in the upstream contrib. But for now, constant is fine